.github/workflows/maven_release.yml

```diff
@@ -79,7 +66,7 @@ jobs:
           server-password: MAVEN_PASSWORD
 
       - name: Publish package
-        run: mvn -DskipTests=true --batch-mode -P ossrh-publish -Dgpg.passphrase=${{ secrets.ORG_GPG_PASSPHRASE }} deploy
+        run: mvn -Dmaven.test.skip=true --batch-mode -P ossrh-publish -Dgpg.passphrase=${{ secrets.ORG_GPG_PASSPHRASE }} deploy
```

Comment:
`-Dmaven.test.skip=true` will skip compiling and executing tests. `-DskipTests=true` only skips executing tests.